README.md

```diff
@@ -1,27 +1,57 @@
 # Data Catalog Frontend
 
-This is a REACT frontend for [DKAN2](https://github.com/GetDKAN/dkan2).
+This is a REACT frontend designed to utilize the latest stable version of [DKAN 2.x](https://github.com/GetDKAN/dkan) as a backend.
 
-This App serves as a starter App, or example of how to use the [data-catalog-components](https://github.com/GetDKAN/data-catalog-components) library to easily create open data catalogs.
+This application serves as a starter app, or example of how to use the [data-catalog-components](https://github.com/GetDKAN/data-catalog-components) library to easily create open data catalogs.
 
-This app utilizes the latest stable version of Drupal 8's version of [DKAN](https://github.com/GetDKAN/dkan2) as a backend. 
 
+## Auto Set Up
+1) Follow the [DKAN Tools](https://github.com/GetDKAN/dkan-tools) README to stand up the backend. Include the `--demo` flag to have the frontend installed and example pages built as well.
+
+## Manual Set Up
+If you have a backend already running and just need the frontend:
 
-## Using the App
-1) Follow the [DKAN2](https://github.com/GetDKAN/dkan2) README to stand up the backend
 1) Install the Gatsby CLI ``npm install -g gatsby-cli``
-1) Clone this repository
+1) Clone this repository in your docroot
 1) Install the dependencies with [npm](https://www.npmjs.com/):
    1) ``cd data-catalog-frontend``
    1) ``npm install``
 1) Run the server: ``npm start`` or ``gatsby develop``
    1) Your site is now running at ``http://localhost:8000``
+1) Edit the `.env.production` file to change the GATSBY_API_URL to reflect your site url
```

Comment:
This should go before the run the server step. Gatsby will need to know this when it builds all the links.